### DISCUSSION TITLE
Excited for 15's changes! Any ideas on when it'll get to our TVs?

### DISCUSSION THREAD

**Discussion by sbsaylors** [jellyfin-archive/jellyfin-roku-legacy#716] [Q&A] [answered]
Excited for 15's changes! Any ideas on when it'll get to our TVs?  Wondering if the brand or type of roku matters? (I have TCL for example)

**Comment by 1hitsong** [comment, marked as answer]
It was released yesterday ~1pm EST and would begin propagating from there.

**Reply by sbsaylors** [reply]
Thanks! Checked the Roku tv and updated. Boom it's there. Can't say thank you enough for getting that audio added in 15.

**Reply by hoboken411** [reply]
Same question for v1.6.3.... Does the release on Github mean it was simultaneously released on Roku? Or does it take a few days to make it to the Roku ecosystem?
